lib/ios-log.js

```diff
@@ -0,0 +1,203 @@
+import path from 'path';
+import _ from 'lodash';
+import which from 'which';
+import glob from 'glob';
+import { fs, getLogger } from 'appium-logger';
+import mkdirp from 'mkdirp';
+import touch from 'touch';
+import xcode from 'xcode'; 
+import { SubProcess } from 'teen_process';
+
+// Date-Utils: Polyfills for the Date object
+
+let START_TIMEOUT = 10000;
+let logger = getLogger('IOS-log');
+
+class IOSLog { 
+
+  constructor(opts) {
```

Comment:
let's go through a list of these fields and make sure they are actually used, otherwise let's get rid of unused ones